### PR TITLE
openwisp-config: update to 1.0.1

### PR DESCRIPTION
**Pull request by nemesisdesign**
Maintainer: @nemesisdesign
Compile tested: ath79, ramips
Run tested: ath79, ramips

### CODE REVIEW

**Comment by nemesisdesign** [comment]
@feckert there's bugfix release for openwisp-config

File: admin/openwisp-config/Makefile [hunk 1]
```diff
@@ -11,9 +11,9 @@ PKG_MAINTAINER:=Federico Capoano <f.capoano@openwisp.io>
 PKG_LICENSE:=GPL3.0-or-later
 
 PKG_SOURCE_URL:=https://github.com/openwisp/openwisp-config.git
-PKG_MIRROR_HASH:=f0b0866b27a7a75bc33c3bbc4780dd99f5ae693226012ae04e63a303172c2c38
+PKG_MIRROR_HASH:=7daa10a9d170e665f33a5555a246b4da2223c2d8d0e8a047edb01701c8886986
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_VERSION:=1.0.0
+PKG_SOURCE_VERSION:=1.0.1
 PKGARCH:=all
```
Comment (BKPepe) [inline admin/openwisp-config/Makefile:17]: Nitpick: This or the row at 32 should be removed as you are stating it twice.

**Review by nemesisdesign — COMMENTED** [review]
Thanks @BKPepe, I removed the first `PKGARCH` since there's only one package.

There's also another PR for OpenWISP Monitoring waiting here: https://github.com/openwrt/packages/pull/18510.

**Comment by feckert** [comment]
Thanks done

**Pull request merged**